backend/tailwind.config.ts

```diff
@@ -57,11 +57,11 @@ module.exports = {
             },
             keyframes: {
                 floating: {
-                    '0%': { transform: 'translate(0, 0px)' },
-                    '50%': { transform: 'translate(0, 1rem)' },
-                    '100%': { transform: 'translate(0, -0px)' },
-                },
-            },
+                    "0%": { transform: "translate(0, 0px)" },
+                    "50%": { transform: "translate(0, 1rem)" },
+                    "100%": { transform: "translate(0, -0px)" }
+                }
+            }
```

Comment:
WTF is this, Dont modify it like this?

Reply:
We want floating animation, not bouncing...